browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -2138,11 +2147,17 @@ index 0c442ee908877d2067d65e87441652a8cf00c88e..6fb448784e0d97fe60eb233853905138
 +        return networkAgent->interceptRequest(loader, WTFMove(handler));
 +    return false;
 +}
++
++void InspectorInstrumentation::willSendEventSourceRequestImpl(InstrumentingAgents& instrumentingAgents)
++{
++    if (InspectorNetworkAgent* networkAgent = instrumentingAgents.inspectorNetworkAgent())
++        networkAgent->willSendEventSourceRequest();
```

Comment:
This is brittle, what if willSendRequest is not called synchronously right after this? Maybe there is a way to pass initiator from the loader that creates request?